__tests__/creatingApp.test.js

```diff
@@ -125,6 +125,102 @@ describe("/api", () => {
         });
       });
    });
+
+   test("404-GET-api/reviews/:review_id/comments", () => {
+    return request(app)
+    .get('/api/reviews/100000/comment')
+    .expect(404);
+  });
+
+  it("400-GET-api/reviews/:review_id/comments", () => {
+    return request(app)
+    .get('/api/reviews/one/comments')
+    .expect(400)
+  });
+ });
+
+  describe('POST-/api/reviews/:review_id/comments', () => {
+    it("POST-201, responds with table of only username and body", () => {
+      const commentToSend = 
+      {
+        userName: "bainesface",
+        body: 'I enjoy coding?'
+      };
+      
+      return request(app)
+      .post("/api/reviews/3/comments")
+      .send(commentToSend)
+      .expect(201)
+      .then((response) => {
+
+      const receivedComment = response.body.comment
+
+      expect(receivedComment).toMatchObject(
+        {
+          author: 'bainesface',
+          votes: expect.any(Number),
+          body: 'I enjoy coding?',
+          review_id: 3,
+          created_at: expect.any(String)
+        }
+      )
+
+      });
+    });
+
+    test("404-POST-api/reviews/:review_id/comments", () => {
```

Comment:
Please change this description to say what the test is testing for eg which status code and why, like above